examples/pytorch/graphsage/model.py

```diff
@@ -93,9 +93,9 @@ def query(self, graph: Graph, inputs: np.ndarray) -> dict:
             n0_out, np.array([[self.feature_idx, self.feature_dim]]), self.feature_type
         )
 
-        context["x0"] = x0
-        context["out_1"] = n1_out.shape[0]  # Number of output nodes of layer 1
-        context["out_2"] = n2_out.shape[0]  # Number of output nodes of layer 2
+        context["x0"] = x0.reshape((context["inputs"].shape[0], -1, self.feature_dim))
+        context["out_1"] =  np.array([n1_out.shape[0]] * context["inputs"].shape[0])  # Number of output nodes of layer 1
```

Comment:
why do you need to create arrays instead of passing shapes?

Reply:
I need to convert from an int to an array with size =batch_size in order to serialze with ray